PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+
+## Definitions
+
+This section provides definitions of the key aspects of this methodology.
+
+### Normative language
+
+The following terminology is used throughout this document to outline requirements and guidelines (accordingly with common standardisation practices):
+- **shall** indicates a *requirement*,
+- **should** indicates a *recommendation*,
+- **may** indicates an *option* that is permitted.
+
+### Acronyms
+
+*(this section shall contains the list of acronyms in alphabetical order)*
+
+| Acronym | Full text           |
+|---------|---------------------|
+| CDM     | Clean development mechanism |
+| EAC     | Energy Attribute Certificate |
+| GHG     | Greenhouse gas(es)  |
+| PV      | Photovoltaic        |
+| UN      | United Nations      |
+
+### Glossary
+
+- **Digital certificate** is a generic term designating environmental attributes in digital form generated using this methodology. Examples of these include EACs and carbon credits. 
+
+- **Emission intensity** (or *carbon intensity*) is a measure of how clean grid electricity is. It refers to how many grams of carbon dioxide equivalent (gCO2e) are released to produce a kilowatt hour (kWh) of electricity. In this methodology, an *average* measure of this factor *on the demand side* is considered at every hour and for a given grid area, meaning that this number reflects the carbon intensity of the *average kilowatt-hour consumed* in this given hour and grid region.
+
+- **High resolution data** is data whose time resolution is hourly or sub-hourly (e.g. every 15 or 30 minutes).
+
+- **Grid region** is a generic term used here to refer to a specific sub-division of a large-scale power system. This may be a balancing region, a country-wide grid, a market-bidding region depending on the local context. The spatial resolution used for the grid region used in a given application should be as granular as allowed by the data available.
+
+- **PV asset** is a set of photovoltaic arrays and associated inverter(s) converting solar irradiance to electrical power.   
+
+## Applicability conditions
+
+This section lays out the requirements on a given context for this methodology to be applicable.
+
+### System eligibility requirements
+
+The system to which the methodology is applied shall meet the following requirements to be eligible:
+
+1. the rating of the PV is between 30 kWp and 5 MWp
+
+2. the system is a fixed installation, meaning that the PV system is mounted on a unique specific site where it will operate throughout the project
+
+3. generation is measured by the inverter(s) or a digital meter with high resolution data, which can either be communicated automatically by digital means (e.g. Rest API) or retrieved manually
+
+4. the asset owner documents the following:
+
+    a. PV asset type and brand
+
+    b. location of the installation (address, or GPS coordinates if an address is not available)
+
+    c. date of installation
```

Comment:
No idea if this would be significant, or if it's just unnecessarily pedantic (!), but should "installation" be more specific? eg. it could be physically installed one day, but not fully operational until a later date. Date on which high frequency data generation commences?

Reply:
Good point, that phrasing was not sharp enough. It is now updated (as well as a few other places for consistency) [here](https://github.com/wattcarbon/WEATS/pull/2/commits/e51d3fc848839fb745c7c3f3bd4041d307d7fede)